lib/zora/filterCasts.ts

```diff
@@ -0,0 +1,43 @@
+import { Cast } from '@neynar/nodejs-sdk/build/neynar-api/v2';
+import getCastsMetadataLink from './getCastsMetadataLink';
+import getCastContractMapping from './mapCastToContract';
+
+async function filterZoraFeed(casts: Cast[]) {
+  try {
+    const data = await getCastsMetadataLink(getCastContractMapping(casts));
+
+    const castsMetadata = await Promise.all(data.map(cast => (
+      fetch(`https://cloudflare-ipfs.com/${cast.ipfs}`, { cache: 'force-cache' })
```

Comment:
did you verify your usage of this gateway is valid?

looks invalid to me

```suggestion
      fetch(`https://cloudflare-ipfs.com/ipfs/${cast.ipfs}`, { cache: 'force-cache' })
```